openebs/README.md

```diff
@@ -2,8 +2,8 @@
 
 ## About OpenEBS
 
-OpenEBS is containerized storage for containers. More details on OpenEBS can be found on [OpenEBS project page](https://github.com/openebs/openebs)
-
+OpenEBS is containerized storage for containers. More details on OpenEBS can be found on [OpenEBS project page.](https://github.com/openebs/openebs)
+OpenEBS Kubernetes PV provisioner is using API's exposed by [maya-apiserver](https://github.com/openebs/mayaserver) to perform provision and delete operation.
```

Comment:
Thanks, the readme also contain a storage class and pvc file references and claim process. Would you like to add those as well in the readme ?